server/controllers/ride.js

```diff
@@ -0,0 +1,16 @@
+import database from '../models/ride';
+
+const db = database;
+
+// returns all rides offers
+const getAllRides = (req, res) => {
+  const rides = db.getAllRides();
+  res.status(200).json({
+    message: 'success',
+    Rides: rides,
```

Comment:
`Rides` should be in lowercase as `rides`. Also since you are using es6 you can just do 
```js